schema/model.py

```diff
@@ -637,10 +637,6 @@ class Config:
     package: Package | None = Field(None, description="The package's metadata information")
     dependencies: Dependencies = DependenciesField
     host_dependencies: Dependencies = HostDependenciesField
-    build_dependencies: Dependencies = BuildDependenciesField
```

Comment:
build_dependencies should still be left in for backwards compatibility (similar to host_dependencies).